commands.js

```diff
@@ -876,8 +877,13 @@ var commands = exports.commands = {
 		}
 
 		if (targetUser.confirmed) {
-			var from = targetUser.deconfirm();
-			ResourceMonitor.log("[CrisisMonitor] " + targetUser.name + " was locked by " + user.name + " and demoted from " + from.join(", ") + ".");
+			if (cmd === 'forcelock') {
```

Comment:
You should do the same thing as `forcepromote`, where it refuses to `forcelock` a user that isn't confirmed. Might as well protect against typoes in both directions.